CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing to the SE701 Team 1 Project
+
+Thank you for investing your time in contributing to our project! 
+
+## Code of Conduct
+This project and everyone participating in it is follows our Code of Conduct (found in the root of this repository). By participating, you are also expected to uphold its standards and are subject to its enforcement guidelines.
```

Comment:
"participating in it _is_ follows" 
Nitpick, but is the "is" correct grammar there?

Reply:
Ah yes you're right! I'll fix that up, thank you

Reply:
fixed now